apis/core/v1alpha2/featuregate_types.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FeatureReference refers to a Feature resource and specifies its intended activation state.
+type FeatureReference struct {
+	// Name is the name of the Feature resource, which represents a feature the system offers.
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+	// Activate indicates the activation intent for the feature.
+	Activate bool `json:"activate,omitempty"`
+	// SkipStabilityPolicyValidation lets you skip the policy validation performed when toggling the feature.
+	// Once set to true, cannot be set back to false
+	SkipStabilityPolicyValidation bool `json:"skipStabilityPolicyValidation,omitempty"`
```

Comment:
I'd name this something more explicit; I think someone could set this without understanding the impact.

How about `PermanentlyVoidAllSupportGuaranteesForThisEnvironment`?

Reply:
I think `PermanentlyVoidAllSupportGuaranteesForThisEnvironment` is too wordy, so went with `PermanentlyVoidAllSupportGuarantees`. Let me know what you think.

Reply:
Works for me!